lib/kamal/configuration/docs/configuration.yml

```diff
@@ -176,3 +176,10 @@ logging:
 # Alias configuration, see kamal docs alias:
 aliases:
   ...
+
+# Network
+#
+# The network the application will be attached to.
+#
+# Defaults to kamal:
+network: custom
```

Comment:
The proxy can be used by multiple applications so we shouldn't configure the network it uses in the application's config as that can lead to incompatible configuration.

There is the `kamal proxy boot_config` option for setting proxy level config. That's where it would belong is we wanted to allow it to be changed. But let's leave that for now at least in this PR.

Also the proxy and the application need to be on the same network, so we shouldn't allow them to be changed separately. If we do make the proxy network configurable, the application should just boot in the same network.

Reply:
So I think we should just stick to allowing the accessory networks to be set in this PR.

Reply:
I wanted to achieve the maximum configurability, but what you're saying makes sense. I will update the PR.

Reply:
I reverted all changes except to the accessory configuration.